docs/api/en/core/BufferGeometry.html

```diff
@@ -40,6 +40,23 @@ <h2>Code Example</h2>
 		const mesh = new THREE.Mesh( geometry, material );
 		</code>
 
+		or if "indexed" version:
+
+		<code>
+		const vertices = new Float32Array( [
+			-1.0, -1.0,  1.0, // 0
+			 1.0, -1.0,  1.0, // 1
+			 1.0,  1.0,  1.0, // 2
+			-1.0,  1.0,  1.0, // 3
+		] );
+
+		const indices = new Uint32Array( [
+			0, 1, 2, // triangle 1: "0, 1, 2"
+			2, 3, 0, // triangle 2: "2, 3, 0"
+		] );
+		geometry.setIndex(new THREE.BufferAttribute( indices, 1 ));
```

Comment:
The following line is missing:
```js
geometry.setAttribute( 'position', new THREE.BufferAttribute( vertices, 3 ) );
```

Besides, we do not recommend that user define a typed array for indices since they often pick the wrong type. So it's better to use a normal array and let `setIndex()` decide whether to use `Uint32Array` or `Uint16Array `.

BTW: `Uint16Array ` is sufficient for the above data.

Reply:
> I didn't wanted to repeat this, as it stays unchanged from the previous "non-indexed" version, but I can if you prefer.

Please do. User will copy the code snippet and easily miss the position attribute.